src/transformers/adapters/hub_mixin.py

```diff
@@ -162,54 +162,40 @@ def push_adapter_to_hub(
                 True.
             overwrite_adapter_card (bool, optional): Overwrite an existing adapter card with a newly generated one.
                 If set to `False`, will only generate an adapter card, if none exists. Defaults to False.
+            create_pr (bool, optional):
+                Whether or not to create a PR with the uploaded files or directly commit.
 
         Returns:
             str: The url of the adapter repository on the model hub.
         """
-        repo_url = self._get_repo_url_from_name(
-            repo_name, organization=organization, private=private, use_auth_token=use_auth_token
-        )
-        if local_path is not None:
-            repo_path = local_path
+        if not repo_name.startswith(organization):
```

Comment:
This line throws an exception for me when `organization` is not set.
```suggestion
        if organization is not None and not repo_name.startswith(organization):
```

Maybe we could also think about deprecating the `organization` parameter as done in Transformers?

Reply:
I think deprecating organization to keep the AdapterHub API similar to HuggingFace makes a lot of sense! I just didn't want to change your API as a first time contributor!

Reply:
For now, I've added your suggested change and kept everything backwards compatible. If you want to make this a breaking change and deprecate the organization argument, I can make that shift instead!

Reply:
I think we can go for the change but keep the organization parameter for backward compatibility with a deprecation warning, similar to HF does it here: https://github.com/huggingface/transformers/blob/d1c9242db2092e9d96b1b18ba9fe27f3518184a5/src/transformers/utils/hub.py#L891-L899

Reply:
Hey @Helw150, are you working on the deprecation warning implementation suggested by @calpt? We are planning to release the next AdapterHub version next week and would like to include your pull request.

Reply:
Sorry - hadn't seen this notification. Just added the warning!